src/python_routing_v02/fast_reach/mc_reach.pyx

```diff
@@ -56,7 +56,7 @@ cpdef object binary_find(object arr, object els):
 
 
 @cython.boundscheck(False)
-cdef void compute_reach_kernel(float qup, float quc, int nreach, const float[:,:] input_buf, float[:, :] output_buf, bint assume_short_ts) nogil:
+cdef void compute_reach_kernel(float qup, float quc, int nreach, const float[:,:] input_buf, float[:, :] output_buf, bint assume_short_ts, bint return_courant=False) nogil:
```

Comment:
New argument, `return_courant` added to `compute_reach_kernel`. If true, courant metrics are added to `output_buf` and returned from the function